lib/upipe/uprobe_stdio.c

```diff
@@ -238,6 +262,18 @@ void uprobe_stdio_set_color(struct uprobe *uprobe, bool enabled)
     uprobe_stdio->colored = enabled;
 }
 
+/** @This sets the output time format or disables it.
+ *
+ * @param uprobe pointer to probe
+ * @param format strftime format or NULL to disable
+ */
+void uprobe_stdio_set_time_format(struct uprobe *uprobe, const char *format)
+{
+    struct uprobe_stdio *uprobe_stdio = uprobe_stdio_from_uprobe(uprobe);
+    free(uprobe_stdio->time_format);
+    uprobe_stdio->time_format = format ? strdup(format) : NULL;
+}
+
```

Comment:
Perhaps it would be more appropriate to return `UBASE_ERR_ALLOC` if `strdup` fails?